Fix: add duration to status for audio files

Technici4n opened the pull request:
Without this, the duration in the *status* object is 0 for audio files. Now the correct duration is displayed. For video files, it shouldn't matter anyways.

Technici4n committed:
Fix: add duration to status for audio files

depuits commented on lib/player.js:
```diff
@@ -131,12 +131,17 @@ Player.prototype = _.extend({
-                fullscreen: (parseInt(data.match(/ANS_VO_FULLSCREEN=([01])/)[1]) === 1),
-                subtitles: (parseInt(data.match(/ANS_SUB_VISIBILITY=([01])/)[1]) === 1)
-            });
+        if(data.indexOf('ANS_LENGTH') !== -1 || data.indexOf('ANS_VO_FULLSCREEN') !== -1 || data.indexOf('ANS_SUB_VISIBILITY') !== -1) {
```
This or check seems completly unnesesary because the checks are repeated for each item below

Technici4n replied:
Indeed. Do you want me to remove it ?

depuits replied:
I would remove it but there still needs to be a check for calling the setStatus so it isn't called when nothing is changed

Commit:
Fixed Maps for setOption

Varpars as maps dont have a length value, but you can get the keys and
work with them.

Technici4n committed:
Merge pull request #1 from Riesi/master

Fixed Maps for setOption
